content/docs/lists-and-keys.md

```diff
@@ -202,13 +201,13 @@ ReactDOM.render(
 );
 ```
  
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/ZXeOGM?editors=0010)
+[**Wypróbuj na CodePen**](https://codepen.io/gaearon/pen/ZXeOGM?editors=0010)
  
-A good rule of thumb is that elements inside the `map()` call need keys.
+Podstawową regułą jest to że elementy wewnątrz wywołania `map()` potrzebują kluczy.
  
-### Keys Must Only Be Unique Among Siblings {#keys-must-only-be-unique-among-siblings}
+### Klucze muszą być unikalne tylko spośród rodzeństwa {#keys-must-only-be-unique-among-siblings}
```

Comment:
```suggestion
### Klucze muszą być unikalne tylko wśród rodzeństwa {#keys-must-only-be-unique-among-siblings}
```

Reply:
@jfrej Tutaj zostawiłbym tak jak było. Jak porównasz sobie:
https://sjp.pl/spo%C5%9Br%C3%B3d
z
https://sjp.pl/%C5%9Br%C3%B3d

To na pierwszy rzut oka, `wśród` gorzej leży na końcu zdania niż `spośród`. Dodatkowo `spośród` ma jedno znaczenie, które w tym przypadku bardzo dobrze pasuje.

@jakubdrozdek Jak sądzisz ?

Reply:
Statystyki wystąpień _unikalny spośród_ vs. _unikalny wśród_ w Google "przeważają" w stronę _wśród_. Wydaje mi się, że różnica jest bardzo subtelna. Można by faktycznie doszukiwać się tutaj formy poprawnej, ale przeciętny czytelnik i tak nie zwróci uwagi i się nie przyczepi.

Moim zdaniem _Klucze muszą być unikalne tylko **spośród** rodzeństwa_ brzmi bardziej jakby "klucze" były jednym z elementów zbioru "rodzeństwo" (nasuwa mi się np. _Tylko 20% spośród wyborców zagłosowało na XYZ_ lub _Dwoje spośród trzech osób <coś tam zrobiło>_), co jest oczywiście nielogiczne w tym zdaniu. Natomiast _Klucze muszą być unikalne tylko **wśród** rodzeństwa_ wskazuje bardziej na to, na co ma wskazywać, czyli klucz jest pewną cechą każdego z "rodzeństwa" i musi być unikalny. Ale... może za bardzo mnie poniosło i różnica jest inna ;-) Przynajmniej tak mi się kojarzy, gdy to czytam na głos.

Reply:
Po dłuższym przemyśleniu **wśród** będzie lepsze. Naniosłem zmianę. :)